### DISCUSSION TITLE
Commit message option not always applied

### DISCUSSION THREAD

**Discussion by ahabb1** [renovatebot/renovate#13792] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Describe the bug

I have configured Renovate with the following for commit messages:
```
{
  "extends": [
    ...
    ":semanticCommits",
    ":semanticCommitTypeAll(chore)",
    ":semanticCommitScopeDisabled"
  ],
  ...
  "commitMessageTopic": "{{depName}}",
  "commitMessageExtra": "",
  "commitMessageSuffix": "",
}
```
This should produce commit messages as follows: "chore: update {{depName}}". I noticed that this is not always the case. While it works for most dependencies it does not for some.
Here is an example where Renovate is updating docker tags:
**Actual:** "chore: update gcr.io/google.com/cloudsdktool/cloud-sdk docker tag to v369"
**Expected:** "chore: update gcr.io/google.com/cloudsdktool/cloud-sdk"

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment, marked as answer]
Wrap those configs options in a package rule.

The docker datasource will have some overrides, those can only be overridden again by package rules.

**Comment by ahabb1** [comment]
Adding the options to package rules as mentioned above solved it.
```
"packageRules": [
    {
      "matchDatasources": ["docker"],
      "commitMessageTopic": "{{depName}}",
      "commitMessageExtra": "",
      "commitMessageSuffix": ""
    }
  ]
```
